### DISCUSSION TITLE
Quais projetos brazucas podemos ajudar nesse #hacktoberfest

### DISCUSSION THREAD

**Discussion by wesmelo** [frontendbr/forum#1230] [Discussão]
O [hacktoberfest](https://hacktoberfest.digitalocean.com) começou hoje e eu queria saber quais projetos brazucas possuem a tag do evento para poder contribuir.

**Comment by kvnol** [comment]
Eu já tô mandando PR: https://github.com/frontendbr/open-source/issues

**Comment by wesmelo** [comment]
@kvnol mas só são válidos o que tem a tag do hacktober na issue do projeto

**Comment by luizbills** [comment]
@wesmelo qualquer projeto é válido, desde que você não infrija as regras (no site fala quais). A tag do hacktoberfest é apenas um meio de incentivar a contribuição nesse mês.

**Comment by felipefialho** [comment]
Como podemos ajudar a deixar os projetos divulgados aqui válidos?

**Comment by luizbills** [comment]
@LFeh todos os projetos teoricamente são válidos, quem invalida é a pessoa que quer contribuir. Ex: o cara criar 5 PRs corrigindo indentação (ao invés de enviar tudo como 1 PR)... no geral, apenas precisam evitar "spam", que qualquer PR é válido (nem precisa do merge ser aceito).

**Comment by hectorgrecco** [comment]
https://github.com/frontendbr/open-source/issues/69

**Comment by lucasferreiralimax** [comment]
Projeto para quem quiser colaborar [Currículo Grátis](https://github.com/frontendbr/open-source/issues/71) :v: from the slums of Shaolin :punch:

**Comment by evertonthepaula** [comment]
> [frontendbr/open-source#69](https://github.com/frontendbr/open-source/issues/69)

Esse projeto parece bem promissor, acredito que precisa de mais uma pitada de organização nas idéias!

**Comment by praganavictor** [comment]
> Projeto para quem quiser colaborar [Currículo Grátis](https://github.com/frontendbr/open-source/issues/71) ✌️ from the slums of Shaolin 👊

estou estudando vue acho que vou tentar ajudar esse projeto

**Comment by TiagoDanin** [comment]
[Aqui](https://github.com/search?q=label%3Ahacktoberfest+state%3Aopen+type%3Aissue+user%3ATiagoDanin&type=Issues) tem uma lista de projetos meu que eu abrir issues para #Hacktoberfest
